### ISSUE TITLE
[BUG/ISSUE] WingetUI installer tries to install VCRedist 2015-2022 x64 even though it's already present

### ISSUE THREAD

**Issue by o-l-a-v** [marticliment/WingetUI#649]
## Before submitting the issue

- [x] **I have checked that the bug is not a duplicate or, if it is, it needs to be reopened**
- [x] **I have checked that I am running the latest version of WingetUI**
- [x] **I have tried resetting and reinstalling WingetUI and the issue persisted or happened again**
- [x] **I have checked the [FAQ](https://github.com/martinet101/WingetUI#faq) section for solutions**
- [x] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#issues) and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**

## Describe the bug

Downloaded WingetUI.Installer.exe v1.6.0 from GitHUb, have never had it installed earlier.
I already have VCRedist 2015-2022 x64 installed, but at the end of the install process of WingetUI, it prompts for installing this dependency.

<details><summary>winget list vcredist</summary>
```
C:\Users\olavb>winget list vcredist
Name                                                               Id                           Version       Source
---------------------------------------------------------------------------------------------------------------------
Microsoft Visual C++ 2013 Redistributable (x86) - 12.0.40664       Microsoft.VCRedist.2013.x86  12.0.40664.0  winget
Microsoft Visual C++ 2015-2022 Redistributable (x86) - 14.34.31931 Microsoft.VCRedist.2015+.x86 14.34.31931.0 winget
Microsoft Visual C++ 2015-2022 Redistributable (x64) - 14.34.31931 Microsoft.VCRedist.2015+.x64 14.34.31931.0 winget
Microsoft Visual C++ 2010  x64 Redistributable - 10.0.40219        Microsoft.VCRedist.2010.x86  10.0.40219    winget
```

</details>

Bug/feature request: Improve detection of the VCRedist dependency, don't ask to install if it it's already there.

## To Reproduce

1. `winget install --id Microsoft.VCRedist.2015+.x64`
2. Download and install WingetUI
3. At the end, the WingetUI installer prompts to install VCRedist 2015-2022 x64

## Expected behaviour

WingetUI installer should not try to install a dependency that's already installed.

**Comment by marticliment** [comment]
yes, but this installer already decrets whether vcredist is installed or not, and does nothing if so.

wingetui requires vcredist to work, so this is expected to happen.

Note that the installers always have the latest version, so what could have happened is that too had an older version, and wingetui just updated it to the latest.

**Comment by o-l-a-v** [comment]
Well, it also prompts for admin. WIngetUI itself installs to `%LOCALAPPDATA%`, and does not require admin permissions.

One more problematic thing with this: With Intune, one can only target either SYSTEM with admin permissions, or CURRENT USER without admin permissions. With WingetUI, one could solve this by first installing VCRedist as SYSTEM, then have WingetUI in it's own Win32 package as CURRENT USER, depending on VCRedist. The way WingetUI installer works now, this would fail, because the WingetUI installer would prompt for admin at the end. Even though VCRedist is installed.

Suggested fix: See if any version of VCRedist v14 x64 is installed. WingetUI can later prompt to update it to the latest version. :)

**Comment by ppvnf** [comment]
I believe this was fixed with 1.6.2

**Comment by marticliment** [comment]
Yes, it was. I forgot to post that here
